public/locales/pl/kalambury.json

```diff
@@ -1,58 +1,58 @@
 {
-  "game": {
-    "name": "Kalambury"
-  },
-  "tools": {
-    "undo": "Usuń ostatnią linię",
-    "clear": "Wyczyść rysunek",
-    "forfeit": "Poddaj się :("
-  },
-  "header": {
-    "wait": "Oczekiwanie na innych graczy",
-    "play": "Narysuj podane hasło"
-  },
-  "subheader": {
-    "wait": {
-      "manage": "Możesz rozpocząć grę w dowolnym momencie",
-      "wait": "Administrator pokoju może rozpocząć grę"
-    },
-    "play": {
-      "guess": "Co jest na rysunku?"
-    }
-  },
   "board": {
     "game": {
-      "new_phrase": "Wymień hasło"
+      "new_phrase": "Wymie\u0144 has\u0142o"
```

Comment:
Why?

Reply:
what?

Reply:
I think this could be enforced by JSON standard 👀 Definitely not pretty. I will remove the step and take a look at husky today ✅

Reply:
> The JSON spec requires UTF-8 support by decoders. As a result, all JSON decoders can handle UTF-8 just as well as they can handle the numeric escape sequences. This is also the case for Javascript interpreters, which means JSONP will handle the UTF-8 encoded JSON as well.

Source: Some random stackoverflow answer